pkg/proto/types.go

```diff
@@ -3062,6 +3062,18 @@ func (s *ScriptInfo) ToProtobuf() *pb.ScriptData {
 	}
 }
 
+func (s *ScriptInfo) ToScriptResponseProtobuf() *pb.ScriptResponse {
+	if s == nil {
+		return &pb.ScriptResponse{}
+	}
+	return &pb.ScriptResponse{
+		ScriptBytes: s.Bytes,
+		ScriptText:  base64.StdEncoding.EncodeToString(s.Bytes),
```

Comment:
Are you sure that `ScriptText` field is base64 encoded data?

Reply:
Similar to ScriptData

Reply:
Okay, because this is a deprecated field